tests/lit-tests/const_shuffle-1.ispc

```diff
@@ -0,0 +1,101 @@
+// This test fails on avx512-x64 targets.
```

Comment:
Why does this fail? is it not-yet implemented?

Reply:
An InstCombine bug (or not yet implemented for 64 vectors)

Reply:
Let's mention this explicitly. If there's an issue, please put it here. So this XFAIL will be more actionable.

Reply:
Do we have an issue filed on InstCombine?

Reply:
Not yet